app/models/concerns/media_object_intercom.rb

```diff
@@ -0,0 +1,21 @@
+# Copyright 2011-2017, The Trustees of Indiana University and Northwestern
+#   University.  Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+#   under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+#   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+#   specific language governing permissions and limitations under the License.
+# ---  END LICENSE_HEADER BLOCK  ---
+
+module MediaObjectIntercom
+  def to_api_json
```

Comment:
Could there be a more specific name for this method?  Also I would hope that it would match the naming in `MasterFileIntercom`.